.github/workflows/test_agents.yaml

```diff
@@ -0,0 +1,34 @@
+on:
+  pull_request: {}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - extra: ""
+            script: "test_basic.py"
+            python-version: "3.10"
+          - extra: "stable-baselines"
+            script: "test_sb.py"
+            python-version: "3.7"
+          - extra: "stable-baselines3"
+            script: "test_sb3.py"
+            python-version: "3.10"
+          - extra: "ray-rllib"
+            script: "test_ray_rllib.py"
+            python-version: "3.10"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{matrix.python-version}}
+      - run: python3 -m pip install wheel==0.38.4
+      - run: python3 -m pip install .[tests]
+      - if: ${{ matrix.extra != '' }}
+        run: python3 -m pip install .[${{matrix.extra}}]
+      - if: ${{ matrix.extra == 'stable-baselines' }}
+        run: python3 -m pip install tensorflow==1.14.0
```

Comment:
@discordianfish tensorflow installation is needed for SB. I recall the problems of SB and why they were not adding tensorflow installation in their dependencies, see:
- https://stable-baselines.readthedocs.io/en/master/guide/install.html#stable-release (see note at the end of the paragraph)
- https://github.com/hill-a/stable-baselines/issues/849

But I am tempted to add tensorflow (the CPU version) as a dependency in our setup.py when installing arena with SB. It is also deprecated (and I will not develop anything anymore on it), but it would simplify our scripts, etc..

Let me know

Reply:
What if people want to use the gpu version? I think that is why we didn't include it, right? But if you feel it makes sense, go for it.

Reply:
Yeah, let's keep it as it is. Thanks for the feedback!